pkg/cmd/pr/merge/merge.go

```diff
@@ -55,8 +55,8 @@ func NewCmdMerge(f *cmdutil.Factory, runF func(*MergeOptions) error) *cobra.Comm
 		Long: heredoc.Doc(`
 			Merge a pull request on GitHub.
 
-			By default, the head branch of the pull request will get deleted on both remote and local repositories.
-			To retain the branch, use '--delete-branch=false'.
+			By default, the head branch of the pull request will not get deleted on both remote and local repositories.
+			To delete the branch, use '--delete-branch=true'.
```

Comment:
This note is no longer necessary since the default is no longer surprising. The regular `--delete-branch` flag documentation below will be enough to teach people about this option. I think you can just remove the whole paragraph. 🙇